<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>persian</class>
 <widget class="QMainWindow" name="persian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bestsync</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="2" column="2" rowspan="2" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_102">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <widget class="QTreeView" name="treeView_2"/>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="toolButton_7">
        <property name="toolTip">
         <string>تنظیمات مقایسه</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../settings (1).png</normaloff>../settings (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="toolButton_10">
        <property name="toolTip">
         <string>جا به جایی</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../swap.png</normaloff>../swap.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton_4">
        <property name="toolTip">
         <string>همسان سازی از راست به چپ</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../left.jpg</normaloff>../left.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_93">
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>عملیات بعد از یک ساعت</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string extracomment="File time &amp; size">مقایسه</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../file.png</normaloff>../file.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="5">
       <layout class="QHBoxLayout" name="horizontalLayout_94">
        <item>
         <widget class="QComboBox" name="comboBox_2"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>جستجو</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_96">
        <item>
         <widget class="QToolButton" name="toolButton_8">
          <property name="toolTip">
           <string>فیلتر</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../funnel.png</normaloff>../funnel.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_6">
          <property name="toolTip">
           <string>تنظیمات یکسان سازی</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../settings.png</normaloff>../settings.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="5">
       <layout class="QHBoxLayout" name="horizontalLayout_95">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>همسان سازی</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../sync.png</normaloff>../sync.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButton_5">
        <property name="toolTip">
         <string>همسان سازی از چپ به راست</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../right.jpg</normaloff>../right.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_92">
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>جستجو</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>پوشه</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="actionSave_as_batch_job_2"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>عملیات</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="separator"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>ابزار</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>زبان</string>
     </property>
     <addaction name="action"/>
    </widget>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>کمک</string>
    </property>
    <addaction name="action_22"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="icon">
    <iconset>
     <normaloff>../new.png</normaloff>../new.png</iconset>
   </property>
   <property name="text">
    <string>جدید</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset>
     <normaloff>../open-file-button.png</normaloff>../open-file-button.png</iconset>
   </property>
   <property name="text">
    <string>باز کردن</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset>
     <normaloff>../save.png</normaloff>../save.png</iconset>
   </property>
   <property name="text">
    <string>ذخیره</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>ذخیره به عنوان</string>
   </property>
  </action>
  <action name="actionSave_as_batch_job_2">
   <property name="text">
    <string>Save as batch job</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>خروج</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset>
     <normaloff>../file.png</normaloff>../file.png</iconset>
   </property>
   <property name="text">
    <string>شروع مقایسه</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_13">
   <property name="icon">
    <iconset>
     <normaloff>../settings (1).png</normaloff>../settings (1).png</iconset>
   </property>
   <property name="text">
    <string>تنظیمات مقایسه</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_14">
   <property name="icon">
    <iconset>
     <normaloff>../funnel.png</normaloff>../funnel.png</iconset>
   </property>
   <property name="text">
    <string>تنظیمات فیلتر</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset>
     <normaloff>../settings.png</normaloff>../settings.png</iconset>
   </property>
   <property name="text">
    <string>تنظیمات یکسان سازی</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="action_17">
   <property name="icon">
    <iconset>
     <normaloff>../sync.png</normaloff>../sync.png</iconset>
   </property>
   <property name="text">
    <string>شروع یکسان سازی</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_18">
   <property name="icon">
    <iconset>
     <normaloff>../binoculars.png</normaloff>../binoculars.png</iconset>
   </property>
   <property name="text">
    <string>یافتن...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>صدور لیست</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>../united-kingdom.png</normaloff>../united-kingdom.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_22">
   <property name="icon">
    <iconset>
     <normaloff>../faq.png</normaloff>../faq.png</iconset>
   </property>
   <property name="text">
    <string>مشاهده کمک</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
